jack.html

```diff
@@ -0,0 +1,252 @@
+                    </div><!--//meta-->
+                    <div class="details">
+                        <p>Vanwege zijn brede kennis en ervaring is Jack gevraagd om binnen een kernteam van specialisten te werken aan het verbeteren van de performance van een grote HR-applicatie. Binnen het team, bestaande uit Architect, DBA-ers, en een andere developer hebben zij op allerlei vlakken van het applicatiedomein verbeteringen kunnen doorvoeren.</p>
+						<p>Jack heeft samen met zijn collega's alle applicatie-onderdelen geanaliseerd en bepaald waar verbeteringen de meeste waarde hadden. Metingen en analyses heeft hij gedocumenteerd, zodat dit voor andere collega's binnen de afdeling zcihtbaar zijn en later binnen de organisatie kunnen worden opgepakt. Cruciale verbeteringen heeft hij met zijn team zelf geimplementeerd. De analyse en oplossing zijn in dat geval uitgebreid gedocumenteeerd en gepresenteerd en dienen als kennisoverdacht voor de collega's. Hij heeft zich op alle gebieden ingezet: van front-end caching, server side caching tot en met Database query optimalisaties.
+                        </p>
+                    </div><!--//details-->
+                </div><!--//item-->
+                
+                <div class="item">
+                    <div class="meta">
+                        <div class="upper-row">
+                            <h3 class="job-title">Senior Developer / Techical en Functional lead</h3>
+                            <div class="time"><i class="fa fa-calendar"></i> 2015 - 2016</div>
+                        </div><!--//upper-row-->
+                        <div class="company">Raet B.V.</div>
+                    </div><!--//meta-->
+                    <div class="details">
+                        <p>In zijn rol als lead heeft een nieuw offshore team, bestaande uit 8-10 personen in Bangalore (India) ingewerkt en aangestuurd. Samen met het team heeft hij het product in kaart gebracht met behulp van de C4-methode. Vanuit de door het team verkregen inzichten heeft hij samen met hen de applicatie door ontwikkeld.</p>
+						<p>
+						Vanwege zijn vergaande functionele en technische kennis heeft Jack gedurende deze jaren ook veel bezig gehouden met 3e-lijns activteiten. Hij heeft samen met het Service Center, de consultants en wanneer nodig zelfs met de klant, gewerkt aan het komen tot oplossingen. Wanneer nodig heeft hij ook de taken van de Product Owner waargenomen en regematig geassisteerd in het uitvoeren van probleemanalyses of het opstellen van release notes.</p>
+                    </div><!--//details-->
+                </div><!--//item-->
+                
+                <div class="item">
+                    <div class="meta">
+                        <div class="upper-row">
+                            <h3 class="job-title">Senior Developer</h3>
+                            <div class="time"><i class="fa fa-calendar"></i> 2011 - 2014</div>
```

Comment:
Wat mij betreft laten we deze en de volgende opdrachten vervallen. We proberen het online profiel zo kort en bondig mogelijk te houden.